src/dotnet/commands/dotnet-cache/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Tools.MSBuild;
+
+namespace Microsoft.DotNet.Tools.Cache
+{
+    public partial class CacheCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false);
+            app.Name = "dotnet cache";
+            app.FullName = LocalizableStrings.AppFullName;
+            app.Description = LocalizableStrings.AppDescription;
+            app.AllowArgumentSeparator = true;
+            app.ArgumentSeparatorHelpText = HelpMessageStrings.MSBuildAdditionalArgsHelpText;
+            app.HelpOption("-h|--help");
+
+            CommandOption projectArgument = app.Option(
+                $"-e|--entries <{LocalizableStrings.ProjectEntries}>", LocalizableStrings.ProjectEntryDescription,
+                CommandOptionType.SingleValue);
+
+            CommandOption frameworkOption = app.Option(
+                $"-f|--framework <{LocalizableStrings.FrameworkOption}>", LocalizableStrings.FrameworkOptionDescription,
+                CommandOptionType.SingleValue);
+
+            CommandOption runtimeOption = app.Option(
+                $"-r|--runtime <{LocalizableStrings.RuntimeOption}>", LocalizableStrings.RuntimeOptionDescription,
+                CommandOptionType.SingleValue);
+
+            CommandOption outputOption = app.Option(
+                $"-o|--output <{LocalizableStrings.OutputOption}>", LocalizableStrings.OutputOptionDescription,
+                CommandOptionType.SingleValue);
+
+            CommandOption fxOption = app.Option(
+                $"--framework-version <{LocalizableStrings.FrameworkVersionOption}>", LocalizableStrings.FrameworkVersionOptionDescription,
+                CommandOptionType.SingleValue);
+
+            CommandOption optimizationOption = app.Option(
+                $"--skipoptimization <{LocalizableStrings.SkipOptmizationOption}>", LocalizableStrings.SkipOptmizationOptionDescription,
```

Comment:
@ramarag the long options are separated by a "-". So, this should become `--skip-optimization`.